packages/client/mutations/handlers/handleRemoveTeams.ts

```diff
@@ -5,6 +5,7 @@ import pluralizeHandler from './pluralizeHandler'
 const handleRemoveTeam = (teamId: string, store: RecordSourceSelectorProxy<any>) => {
   const viewer = store.getRoot().getLinkedRecord('viewer')!
   safeRemoveNodeFromArray(teamId, viewer, 'teams')
+  viewer.setValue(false, 'canAccess', {entity: 'Team', id: teamId})
```

Comment:
this is interesting! i figured if the team object is missing then that'd trigger a query to the server that would include `canAccess` in the query. how'd you find out that the cache wasn't getting invalidated?

Reply:
It does, but not fast enough. If I remove the `&& !team` part in the checks `if (!canAccessTeam && !team)` the redirect happens before the refetch. So this is a bit redundant, but I like it better than possibly running into issues later.